jobs/delorean/jenkinsfiles/branch/suite-ga/Jenkinsfile

```diff
@@ -1,115 +1,135 @@
 #!groovy
 
+//https://github.com/comquent/imperative-when
+import org.jenkinsci.plugins.pipeline.modeldefinition.Utils
+
+def when(boolean condition, body) {
```

Comment:
We can import the delorean shared pipeline library here so we don't have to define the helper functions here anymore. `@Library('delorean-pipeline-library')`